tests/utils.go

```diff
@@ -359,6 +359,49 @@ func getPodsByLabel(label, labelType, namespace string) (*k8sv1.PodList, error)
 	return pods, nil
 }
 
+func GetProcessName(pod *k8sv1.Pod, pid string) (output string, err error) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return
+	}
+
+	fPath := "/proc/" + pid + "/comm"
+	output, err = exec.ExecuteCommandOnPod(
+		virtClient,
+		pod,
+		"compute",
+		[]string{"cat", fPath},
+	)
+
+	return
+}
+
+func ListCgroupThreads(pod *k8sv1.Pod) (output string, err error) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return
+	}
+
+	output, err = exec.ExecuteCommandOnPod(
+		virtClient,
+		pod,
+		"compute",
+		[]string{"cat", "/sys/fs/cgroup/cpuset/tasks"},
+	)
+
+	if err == nil {
+		// Cgroup V1
+		return
+	}
+	output, err = exec.ExecuteCommandOnPod(
+		virtClient,
+		pod,
+		"compute",
+		[]string{"cat", "/sys/fs/cgroup/cgroup.threads"},
+	)
+	return
+}
```

Comment:
Let me suggest the following:
```go
func ListCgroupThreads(pod *k8sv1.Pod) (output string, err error) {
	virtClient, err := kubecli.GetKubevirtClient()
	if err != nil {
		return
	}

	threadsFilePath := "/sys/fs/cgroup/"
	if IsCgroup2UnifiedMode() {
		threadsFilePath = filepath.Join(threadsFilePath, "cgroup.threads")
	} else {
		threadsFilePath = filepath.Join(threadsFilePath, "cpuset", "tasks")
	}

	output, err = exec.ExecuteCommandOnPod(
		virtClient,
		pod,
		"compute",
		[]string{"cat", threadsFilePath},
	)

	return
}
```

I think it would be both clearer and safer, WDYT?